### PR TITLE
OBSDOCS-327: Fix logging kafka docs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OBSDOCS-327

Link to docs preview:
https://66220--docspreview.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/log-forwarding#cluster-logging-collector-log-forward-kafka_log-forwarding

QE review:
- [x] QE has approved this change.

Additional information:
I believe that this module was removed by mistake. Need confirmation from @rolfedh before merging in case there was a reason we're not aware of why this was removed.

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: modules/cluster-logging-collector-log-forward-kafka.adoc [hunk 1]
```diff
@@ -91,15 +97,15 @@ spec:
...
-<3> Use the `topic` key to specify the target topic that will receive the logs.
+<3> Use the `topic` key to specify the target topic that receives the logs.
 
-. Create the CR object:
+. Apply the `ClusterLogForwarder` CR by running the following command:
 +
 [source,terminal]
 ----
-$ oc create -f <file-name>.yaml
+$ oc apply -f <filename>.yaml
```
Comment (bburt-rh) [inline modules/cluster-logging-collector-log-forward-kafka.adoc:110]: Would this need a callout? Someting like: "Replace `<filename>` with the name of the file." Not sure. Maybe it's already obvious, but the other steps in the proc use callouts. Just a thought for you to consider...

**Review by bburt-rh — COMMENTED** [review]
LGTM. Just added a single suggestion for you to think about. Nothing big. Thanks!

File: modules/cluster-logging-collector-log-forward-kafka.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/cluster-logging-collector-log-forward-kafka.adoc:110]: I don't think so. This is a command that's used really widely across products in our docs and it's never used with a callout 🙂

**Comment by QiaolingTang** [comment]
LGTM.

**Pull request merged**